src/Microsoft.Data.Analysis/ArrowStringDataFrameColumn.cs

```diff
@@ -53,11 +66,19 @@ public ArrowStringDataFrameColumn(string name, ReadOnlyMemory<byte> values, Read
         }
 
         private long _nullCount;
+        /// <summary>
+        /// <inheritdoc/>
```

Comment:
I don't think you need `<summary>` around these. I think you simply say:

```C#
/// <inheritdoc/>
public override long NullCount => _nullCount;
```

See [https://github.com/aspnet/Scaffolding/blob/9efe0f7e05e51bb43c878be49087bdba7456e36d/src/VS.Web.CG.Contracts/ProjectModel/CommonProjectContext.cs#L10](https://github.com/aspnet/Scaffolding/blob/9efe0f7e05e51bb43c878be49087bdba7456e36d/src/VS.Web.CG.Contracts/ProjectModel/CommonProjectContext.cs#L10) for an example.

Reply:
Correct, `<summary></summary>` is not required.

Note: I don't think inheritdoc works properly. When I have seen it being used, the xmls we generate in artifacts when we build, end up not including any inherited documentation.
This prevents me from automatically porting documentation to dotnet-api-docs. So my suggestion would be to add proper documentation instead of using inheritdoc.

Reply:
Asp.net uses it. How come it works for them?

It saves a ton of duplication and things getting out of sync, since it is only defined in one spot.